src/docfx/build/markdown/BuildMarkdown.cs

```diff
@@ -28,18 +28,10 @@ public static Task Build(Context context, Document file, TableOfContentsMap tocM
                 },
             };
 
+            context.Report(file, markup.Errors);
```

Comment:
would we return the errors to top level and there report? otherwise the errors can be report everywhere in the pipeline, really hard to debug

Reply:
as we talked offline, please return all errors to build level and build errors just like build documents

---
In reply to: [193959829](https://github.com/dotnet/docfx/pull/2857#discussion_r193959829) [](ancestors = 193959829)

Reply:
I left a TODO to consolidate both Report and WriteJson. This PR is already too big

---
In reply to: [193979951](https://github.com/dotnet/docfx/pull/2857#discussion_r193979951) [](ancestors = 193979951,193959829)